### DISCUSSION TITLE
Can it Play Mkv ?

### DISCUSSION THREAD

**Discussion by geminigeek** [quasarframework/quasar-ui-qmediaplayer#306] [General]
Hi,
can this play .Mkv files, i tried  but only audio is coming in Chrome ?
or for mkv some new props need to be set?

**Comment by hawkeye64** [comment]
The issue with `.mkv` files is that some of them use codecs that are not commonly found on your operating system. This means, as a user, you need to download or license the appropriate codec that's needed for the video and/or audio portions of the mkv file.
